From: Dima Kogan <dima@secretsauce.net>
Date: Thu, 26 Nov 2015 21:04:18 -0800
Subject: removed duplicate setting of _PREFIX variables

---
 backup-manager | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/backup-manager b/backup-manager
index e94019c..97cf507 100755
--- a/backup-manager
+++ b/backup-manager
@@ -34,11 +34,6 @@ BIN_PREFIX=/usr/bin
 LIB_PREFIX=/usr/lib
 VAR_PREFIX=/var/lib
 
-#Set prefix for paths
-BIN_PREFIX=/usr/bin
-LIB_PREFIX=/usr/lib
-VAR_PREFIX=/var/lib
-
 # All the paths we provide
 libdir="$LIB_PREFIX/backup-manager"
 vardir="$VAR_PREFIX/backup-manager"
