## 02_cdrecord_to_wodim.dpatch by  <sukria@>

--- a/lib/externals.sh
+++ b/lib/externals.sh
@@ -10,7 +10,7 @@
 mkisofs=$(which mkisofs 2> /dev/null) || true
 growisofs=$(which growisofs 2> /dev/null) || true
 dvdrwformat=$(which dvd+rw-format 2> /dev/null) || true
-cdrecord=$(which cdrecord 2> /dev/null) || true
+cdrecord=$(which wodim 2> /dev/null) || true
 md5sum=$(which md5sum 2> /dev/null) || true
 bc=$(which bc 2> /dev/null) || true
 mysqldump=$(which mysqldump 2> /dev/null) || true
