## 05_mkisofs_to_genisoimage.dpatch by  <sukria@>

--- a/lib/externals.sh
+++ b/lib/externals.sh
@@ -7,7 +7,7 @@
 dar=$(which dar 2> /dev/null) || true
 tar=$(which tar 2> /dev/null) || true
 rsync=$(which rsync 2> /dev/null) || true
-mkisofs=$(which mkisofs 2> /dev/null) || true
+mkisofs=$(which genisoimage 2> /dev/null) || true
 growisofs=$(which growisofs 2> /dev/null) || true
 dvdrwformat=$(which dvd+rw-format 2> /dev/null) || true
 cdrecord=$(which wodim 2> /dev/null) || true
